Frontend/src/api/RecentPapers.jsx

```
@@ -0,0 +1,103 @@
+
+      const response = await api.post(`/research/request-collaboration/${researchId}/`, formData, {
+        headers: { 
+          "Content-Type": "application/x-www-form-urlencoded"
+        },
+      });
+
+      if (response.status === 200) {
+        alert("Collaboration request sent successfully!");
+        setCollabRequested(true);
+      } else {
+        alert("Error: " + response.data.detail);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)